rskj-core/src/test/java/co/rsk/core/FreeBlock.java

```diff
@@ -0,0 +1,277 @@
+package co.rsk.core;
+
+import co.rsk.trie.Trie;
+import co.rsk.trie.TrieImpl;
+import org.ethereum.core.ImmutableTransaction;
+import org.ethereum.core.Transaction;
+import org.ethereum.util.RLP;
+import org.ethereum.util.RLPElement;
+import org.ethereum.util.RLPList;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+/**
+ * Created by SDL on 12/5/2017.
+ */
+public class FreeBlock {
```

Comment:
Later we should look for a different solution so we don't need to do maintenance work whenever we do changes on Block class or other classes relateds